draft-ietf-opsawg-pcapng.md

```diff
@@ -1463,7 +1463,10 @@ following:
 | 0-1 | Inbound / Outbound packet (00 = information not available, 01 = inbound, 10 = outbound) |
 | 2-4 | Reception type (000 = not specified, 001 = unicast, 010 = multicast, 011 = broadcast, 100 = promiscuous). |
 | 5-8 | FCS length, in octets (0000 if this information is not available). This value overrides the if_fcslen option of the Interface Description Block, and is used with those link layers (e.g. PPP) where the length of the FCS can change during time. |
-| 9-15 | Reserved (MUST be set to zero). |
+| 9 | Csum NotReady, as a consequence the checksum field in the transport header is not valid, and checksum validation on the current packet should not be performed. |
+| 10 | Csum Valid, at least the transport header checksum of the packet has already been validated on the kernel side. If the flag is not set then we are free to check the checksum by ourselves provided that Csum NotReady is also not set. |
```

Comment:
Meaning "the checksum has already been checked on the receive path before it was handed to the packet capture mechanism, so there's no need for the packet analyzer to check it"?

Reply:
right